### ISSUE TITLE
Choices out of order on Firefox

### ISSUE THREAD

**Issue by ggordn3r** [iambateman/RankIt#151]
The order is fine on Chrome and Edge. However, see this poll in Firefox: 
https://rankit.vote/results/ZehVDnsKBEWWvRnIbH8a/summary
https://rankit.skelpo.com/results/ZehVDnsKBEWWvRnIbH8a/summary

Expected behavior is that candidates are in descending order by votes in the first round. Oddly, there seems to be a different order on live vs. staging as well.

I haven't been able to test on Safari. Please check there as well.

![Screenshot (397)](https://user-images.githubusercontent.com/10144772/88857365-2a32f800-d1c4-11ea-9f05-3fe33751a972.png)

**Comment by proggeramlug** [comment]
Hm, this is very odd. I took sorting out, for now, to settle this in issue #130  - let's retest after we are done with that.

**Comment by ggordn3r** [comment]
Now that we've got order settled, I'm looping back on this. 

I'm not seeing results in Firefox at all right now. Tested with these two links, and I see permanent "fetching poll" status. The vote pages work fine, though:
https://rankit.skelpo.com/results/O5z48DDWEe50fsiKXIyw/summary
https://rankit.skelpo.com/results/eaub53HYQM70lX5J8az7/2

The polls are both published with results public and work as expected in Chrome, so it seems like a browser issue.

![image](https://user-images.githubusercontent.com/10144772/91665405-51326180-eac3-11ea-9d7a-870b3057b1a5.png)
